config/puma.rb

```diff
@@ -0,0 +1,21 @@
+# https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server
+
+workers Integer(ENV['WEB_CONCURRENCY'] || 1)
```

Comment:
Doesn't the mem usage depend on what libs you're including and your app code (storing large amounts of data in mem or mem leaks can cause for some bloated memory). It does seem a bit odd that it's using so much memory compared to what @monfresh is seeing with his app.